frontend/packages/ux-editor/src/components/TextResource/TextResource.tsx

```diff
@@ -159,7 +159,7 @@ const TextResourceButton = ({
   onOpen,
   textResourceId,
 }: TextResourceButtonProps) => {
-  const value = textResourceId ? <TextResourceValue id={textResourceId} /> : null;
+  const value = useTextResourceValue(textResourceId);
```

Comment:
When changing this we can delete the component `TextResourceValue` and the test-file for it.

Reply:
Fixed